machine_learning_hep/utilities_validation.py

```diff
@@ -30,6 +30,44 @@ def fillvalidationvsmult(dfevt, dfevtevtsel, df_reco):
     Create a TH3F histogram and fill it with three variables from a dataframe.
```

Comment:
This doesn't look like the proper description of what the function does.

Reply:
I agree, nonetheless, it has nothing to do with this PR, I will fix it though